### PR TITLE
Add rubygem-foreman_resource_quota package

### PR DESCRIPTION
**Pull request by bastian-src**
Adding an rpm build for the [Foreman Resource Quota](https://github.com/ATIX-AG/foreman_resource_quota) plugin.

I've used the `add_gem_package` script and it changed the order of some entries in `comps/comps-foreman-el8` and `comps/comps-foreman-plugin`. I've removed such changes from my commit, but additional verification during the review would be nice.

### CODE REVIEW

File: packages/plugins/rubygem-foreman_resource_quota/rubygem-foreman_resource_quota.spec [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# template: foreman_plugin
+%global gem_name foreman_resource_quota
+%global plugin_name resource_quota
+%global foreman_min_version 3.5.0
```
Comment (sbernhard) [inline packages/plugins/rubygem-foreman_resource_quota/rubygem-foreman_resource_quota.spec:4]: current foreman version is 3.10. should we bump it?

**Review by sbernhard — APPROVED** [review]
LGTM. Thanks.

File: packages/plugins/rubygem-foreman_resource_quota/rubygem-foreman_resource_quota.spec (hunk 1 above)
Comment (evgeni) [reply packages/plugins/rubygem-foreman_resource_quota/rubygem-foreman_resource_quota.spec:4]: This comes from the plugin source. Please update it there if needed.

**Review by evgeni — APPROVED** [review]

**Comment by bastian-src** [comment]
Thanks @evgeni!

**Pull request merged**